app/storybook/documentation/ui-kit/hooks/QueryHooks.mdx

```diff
@@ -0,0 +1,61 @@
+import { Canvas, ArgTypes, Meta, Story, Source, Description, Primary, Controls, Stories } from '@storybook/blocks'
+
+<Meta title="HOOKS/Query Hooks" /> 
+
+# Query Hooks
+
+Propel’s UI Kit library provides the following React components: Counter, TimeSeries, and Leaderboard. To query the GraphQL API, we have developed 
+Query Hooks including "useCounter".
+
+
+## useCounter
+
+```ts  
+import { useCounter } from "@propeldata/ui-kit"  
+import fractionUnicode from 'fraction-unicode'
+
+export default function Fraction(props) {
+  const numerator = useCounter({ query, timeZone})
+  const denominator = useCounter({ query, timeZone})
+
+  return (<p>{fractionUnicode(numerator.value, denominator.value)}</p>)
```

Comment:
I agree. @hasancolak @felipecadavid, please do not use markdown syntax to display code examples. Before [the Theming PR](https://github.com/propeldata/ui-kit/pull/132), we used the [Canvas](https://storybook.js.org/docs/api/doc-block-canvas/) doc block for that purpose. In the PR, I introduced a custom block [SourceCode](https://github.com/propeldata/ui-kit/pull/132/files#diff-da0af4d5ffc9f1c890b6e47d32b334d1005de58763a3920e903d62bcb5f52eb8). Please use it instead of markdown. Also, please avoid hardcoded examples in favor of Stories.